services/web-app/pages/_app.js

```diff
@@ -6,10 +6,10 @@
  */
 import '@/styles/styles.scss'
 
+import { AuthProvider } from 'contexts/auth'
```

Comment:
Did your IDE automatically remove the `@`? This should either have an `@` in front of it or be pulled in from `./contexts/auth` since it's a relative dependency.

Reply:
hmm no idea what happened there, fixed it